README.md

```diff
@@ -1,8 +1,5 @@
 ﻿Working with code owners
 https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-Rebasing
-https://docs.github.com/en/get-started/using-git/about-git-rebase
```

Comment:
this shouldn't have been removed.

Reply:
I removed it by mistake.  Thanks Ill put it back.